portfolio-project-matching-app/pages/myProjects.js

```diff
@@ -35,12 +35,12 @@ const myProjects = () => {
     }, [])
 
     return (
-        <div>
+        <div className='background'>
```

Comment:
utilizing Charlie's design updates to make things more consistent with the rest of the app.